pkg/controllers/controllers.go

```diff
@@ -35,14 +35,15 @@ import (
 	nodeclasstermination "github.com/Azure/karpenter-provider-azure/pkg/controllers/nodeclass/termination"
 
 	"github.com/Azure/karpenter-provider-azure/pkg/controllers/nodeclaim/inplaceupdate"
+	"github.com/Azure/karpenter-provider-azure/pkg/providers/imagefamily"
 	"github.com/Azure/karpenter-provider-azure/pkg/providers/instance"
 )
 
 func NewControllers(ctx context.Context, mgr manager.Manager, kubeClient client.Client, recorder events.Recorder,
-	cloudProvider cloudprovider.CloudProvider, instanceProvider instance.Provider) []controller.Controller {
+	cloudProvider cloudprovider.CloudProvider, instanceProvider instance.Provider, imageProvider *imagefamily.Provider) []controller.Controller {
```

Comment:
minor, style, feel free to ignore:

I think go style guide wants either all args 1 line, or all args on their own line. I know you don't add this split in this PR but you could fix it =)